### ISSUE TITLE
The custom integration midea_ac_lan implements the is_aux_heat

### ISSUE THREAD

**Issue by hpascualargentuminc** [georgezhao2010/midea_ac_lan#511]
### HA core version

2024.5.10

### Intergration version

0.3.22

### Device type and model

44204

### Used App

Msmarthome

### The description of problem

The custom integration midea_ac_lan implements the is_aux_heat property or uses the auxiliary heater methods in a subclass of ClimateEntity.
 This will stop working on HA v2024.10.0

**Comment by 0Styless** [comment]
Thumps up for a deprecation fix.

**Comment by mattlward** [comment]
Agree, thumbs up for a fix.  :)  I use this every summer with my window units.

**Comment by fcastello** [comment]
I have created a PR to remove AUX heating feature. But I don't know how much the maintainer is responding
https://github.com/georgezhao2010/midea_ac_lan/pull/527
Any feedback on the PR welcome. It is untested yet. Follow the PR for more news

Although my fix is to remove the feature not to replace it by something else

**Comment by randyoo** [comment]
This repo is no longer being actively maintained.

There's someone who forked it -- @wuwentao -- and seems to be carrying the torch now: https://github.com/wuwentao/midea_ac_lan

**Comment by hkusulja** [comment]
Duplicate of https://github.com/georgezhao2010/midea_ac_lan/issues/505
please close this

**Comment by hkusulja** [comment]
> The midea_ac_lan custom integration is using deprecated climate auxiliary heater
> This stops working in version 2024.10.0. Please address before upgrading.
> The custom integration midea_ac_lan implements the is_aux_heat property or uses the auxiliary heater methods in a subclass of ClimateEntity.
